forte/data/data_store.py

```diff
@@ -788,7 +815,7 @@ def _add_entry_raw(
         else:
             raise NotImplementedError(
                 "_add_entry_raw() is not implemented "
-                f"for entry type {entry_type}."
+                f"for the parent entry of {type_name}."
```

Comment:
This change makes the error message a bit weird, why emphasize `parent entry` here?